frontend/src/components/common/LookupCodeDropdown.tsx

```diff
@@ -24,26 +24,28 @@ function LookupCodeDropdown(props: LookupCodeDropdownProps) {
   const onCodeItemChange = (eventKey: string) => {
     const eventKeyId = parseInt(eventKey);
     if (eventKeyId < 0) {
-      setCodeDropdownTitle("Any");
+      setCodeDropdownTitle('Any');
       props.onSelectCode(null);
       return;
     }
     const agencyName = getNameById(props.lookupCodes, eventKeyId);
     setCodeDropdownTitle(agencyName);
     props.onSelectCode(eventKeyId);
-  }
+  };
 
   return (
-    <DropdownButton id="dropdown-basic-button" title={codeDropdownTitle} bsPrefix="map-filter-dropdown" onSelect={onCodeItemChange}>
+    <DropdownButton
+      id="dropdown-basic-button"
+      title={codeDropdownTitle}
+      bsPrefix="map-filter-dropdown"
+      onSelect={onCodeItemChange}
+    >
```

Comment:
To review: The above multi-line break is "enforced" by the max-length of lines we set in **prettier.rc** -> `printWidth: 100,`

If we increase that number we can get it to one-line. How does the team feels about 100 chars. Too long? Too short? I know some people have opinions on how readable is code when it is too long but am happy to adapt to whatever the team wants

Reply:
starting with 100 seems fine.